pros/conductor/conductor.py

```diff
@@ -16,27 +17,39 @@
 
 MAINLINE_NAME = 'pros-mainline'
 MAINLINE_URL = 'https://purduesigbots.github.io/pros-mainline/pros-mainline.json'
+BETA_NAME = 'pros-4-beta'
+BETA_URL = 'https://purduesigbots.github.io/pros-mainline/beta/pros-4-beta.json'
```

Comment:
can we name this just `kernel-beta-mainline.json`? just a more futureproof name. idea is we use this for all beta testing on the kernel.